### ISSUE TITLE
Some aria attributes are not read out on DIV with role="combobox"

### ISSUE THREAD

**Issue by dobrinyonkov** [FreedomScientific/VFO-standards-support#481]
When focusing div with role combobox the following list of aria-attributes are not read out:

1. aria-expanded
2. aria-haspopup
3. aria-readonly

Example: 
1. Go to https://jsbin.com/kuhoduduja/edit?html,output
2. Focus the div.

### Expected result
The aria-haspopup attribute to be read out
The expanded state to be read out
The readonly state to be read out

### Actual result
The aria-haspopup attribute is not read out
The expanded state is not read out
The readonly state is not read out

### Examples
1. https://jsbin.com/kuhoduduja/edit?html,output

2. aria-expanded and aria-haspopup issues could also be reproduced in aria practices Select-only combobox example:
https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html

#### JAWS version and build number
2021.2011.16 ILM

#### Operating System and version
Microsoft Windows 10 Enterprise
Version 10.0.18363 Build 18363

#### Browser and version:
Google Chrome Version 87.0.4280.88 (Official Build) (64-bit)

**Comment by JAWS-test** [comment]
I can confirm the problem (with Chrome and JAWS 2020).

But I am unsure if `aria-haspopup` needs to be output. A combobox always has an implicit `aria-haspopup=listbox`. The additional output can be annoying. Moreover, the `aria-haspopup` is also not output at an `input` with `combobox` (https://w3c.github.io/aria-practices/examples/combobox/grid-combo.html).

**Comment by frex65** [comment]
Running JAWS 2021.2012.48,:
* Confirmed that the *aria-haspopup* value is not announce. I agree with @JAWS-test that announcing it in this scenario would be overkill as the user will be expecting a list. IMO this aspect should not be considered a JAWS bug.
* Confirmed that when *aria-readonly* is set to "true", this is not announced in [select-only comboboxes](https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html) such as the one in @dobrinyonkov's example. However it is announced in editable comboboxes. JAWS bug confirmed.

**Comment by PavaniHFP** [comment]
https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/ JAWS 2026 is still not announcing as expanded or collapsed when a user tabs to the control. This bug still exists
